dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/Assertions.java

```diff
@@ -151,7 +167,7 @@ public static void assertHasErrors( ValidationReport report, int count, Validati
      * assertHasWarnings asserts the report contains given count of warnings and
      * warnings that contain given code and that are linked to given tracker
      * object in any order. <em>NOTE:</em> prefer
-     * {@link #assertHasOnlyWarnings(ValidationReport, Warning...)}
+     * {@link #assertHasOnlyWarningsAndNoErrors(ValidationReport, Warning...)}
```

Comment:
something seems of in the javadoc 🤔 are there too many parameters in the `@link`? I assume

`{@link #assertHasOnlyWarningsAndNoErrors(ImportReport, Warning...)}`

is what I meant